fixit.py

```diff
@@ -106,6 +111,16 @@ def valiDate(self,indate,intime):
 
   def getHist(self):
     self.globalMessage.setText("Getting archived values...")
+    if self.checkBoxShowChanged.isChecked():
+      showChanged=1
+    else:
+      showChanged=0
```

Comment:
You could probably one line this like `showChanged = True if self.checkBoxShowChanged.isChecked() else False`

Reply:
Fixed! Thanks!